code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -215,6 +215,7 @@
 		SM.faction = user.faction
 		SM.master_commander = user
```

Comment:
This already logs the master of the sentient animal in a var which any admin can easily check. Adding the same thing to the attack_log seems somewhat redundant.

Also, if a sentient animal manages to reproduce, any offspring inherit the master_commander value, but wouldn't have the new attack log added below.